### DISCUSSION TITLE
while using yarn build:browser command i'm facing this issue. Any idea why it raises & how to resolve it?

### DISCUSSION THREAD

**Discussion by NikshipthYathindra** [eclipse-theia/theia#13651] [Q&A]
```
Error: webpack exited with an unexpected code: 1.
    at ChildProcess.<anonymous> (D:\NextGen\theia\GitAutoApp\node_modules\@theia\application-manager\lib\application-process.js:66:28)
    at ChildProcess.emit (node:events:518:28)
    at maybeClose (node:internal/child_process:1105:16)
    at ChildProcess._handle.onexit (node:internal/child_process:305:5)
error Command failed with exit code 1.
```

**Comment by msujew** [comment]
Hey @NikshipthYathindra,

can you post a bit more of the error message? Just the command failing isn't saying a lot.

**Reply by NikshipthYathindra** [reply]
Hi, Didn't look into the error message until u mentioned, I'm able to fix the issue. Thanks for the response.

**Reply by amiths89** [reply]
@NikshipthYathindra I am facing the same issue. What did u do to resolve this error?

**Reply by msujew** [reply]
@amiths89 Can you post the full error message? The original poster omitted that.

**Reply by amiths89** [reply]
Sorry, my bad. I think I made the same mistake as the person who posted this. The actual error was a few lines above, and i had an import error, which was causing this.

**Comment by nulidexiaoyang** [comment]
can you telle me how to solve it?
i have installed the yarn and add it to my env, but still have the same error

**Reply by nulidexiaoyang** [reply]
i install yarn by npm, maybe question on it ?

**Reply by msujew** [reply]
@nulidexiaoyang Can you post the error message?

**Reply by nulidexiaoyang** [reply]
@msujew thanks,  I have resolved it . the error because of yarn env
